scripts/test_updates_pg12.sh

```diff
@@ -20,3 +20,5 @@ run_tests "$@" -v7 \
 run_tests "$@" -v8 \
           2.5.0-pg12 2.5.1-pg12 2.5.2-pg12 2.6.0-pg12 2.6.1-pg12 2.7.0-pg12 2.7.1-pg12 2.7.2-pg12 \
           2.8.0-pg12 2.8.1-pg12 2.9.0-pg12 2.9.1-pg12 2.9.2-pg12 2.9.3-pg12 2.10.0-pg12 2.10.1-pg12
+run_tests "$@" -r -v8 \
+          2.10.0-pg12 2.10.1-pg12
```

Comment:
```suggestion
          2.8.0-pg12 2.8.1-pg12 2.9.0-pg12 2.9.1-pg12 2.9.2-pg12 2.9.3-pg12
run_tests "$@" -r -v8 \
          2.10.0-pg12 2.10.1-pg12
```
Don't need to run twice tests for 2.10.x versions.